measure_extinction/stardata.py

```diff
@@ -728,8 +736,8 @@ def read_irs(self, line, path="./", corfac=None):
         # self.fluxes *= mfac
         # self.uncs *= mfac
 
-        # correct the IRS spectra if corfacs defined
-        if "IRS" in corfac.keys():
+        # correct the IRS spectra if desired and if corfacs are defined
+        if use_corfac==True and "IRS" in corfac.keys():
```

Comment:
This should just be `if use_corfac and "IRS" in corfac_keys():` as use_corfac is already a boolean.

Reply:
Right! Thanks!